README.md

```diff
@@ -227,7 +227,13 @@ Future work and Limitations
 Change Log (From version 2.2.0 and onwards)
 ==============
 
-### __2.6.0-SNAPSHOT__
+### __3.0.0__
+
+* Upgraded to Hive 1.2.1 (Note: new major release with backwards incompatibility issues). As of Hive 1.2 there are a number of new reserved keywords, see [DDL manual](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Keywords,Non-reservedKeywordsandReservedKeywords) for more information. 
+If you happen to have one of these as an identifier, you could either backtick quote them (e.g. \`date\`, \`timestamp\` or \`update\`) or set hive.support.sql11.reserved.keywords=false.                                            
+* Removed the custom HiveConf hive.vs. Use hadoop.tmp.dir instead.   
```

Comment:
I tested this version of HiveRunner with one of our medium sized projects. It's about 2000 lines of HQL and was build using Hive 0.14. Apart from the reserved keyword issue, I had no problems running the test suite in HiveRunner 3.0.
I would like to just roll forward with one release until we hit some issues that calls for split releases of HiveRunner. We can always at some point create a 0.14 branch and release from that. Is that fine with you?